pkg/rpm/tar.go

```diff
@@ -15,7 +15,7 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-func RPMToTar(rpmReader io.Reader, tarWriter *tar.Writer, noSymlinksAndDirs bool, capabilities map[string][]string, selinuxLabels map[string]string) error {
+func RPMToTar(rpmReader io.Reader, tarWriter *tar.Writer, noSymlinksAndDirs bool, capabilities map[string][]string, selinuxLabels map[string]string, createdPaths map[string]struct{}) error {
```

Comment:
How about making RPMToTar a method on a struct, and keep the createdPaths tracking internal?

Reply:
I finally came around to try this